crates/typst-library/src/math/attach.rs

```diff
@@ -152,5 +152,7 @@ pub struct StretchElem {
 
     /// The size to stretch to, relative to the maximum size of the glyph and
     /// its attachments.
-    pub size: Smart<Rel<Length>>,
+    #[resolve]
+    #[default(Rel::one())]
+    pub size: Rel<Length>,
```

Comment:
Technically a breaking change, but an okay one

Reply:
Yes, I did this in a couple places. It seems quite obvious now that the those don't need to be smart.